packages/mui-material/src/CssBaseline/CssBaseline.js

```diff
@@ -71,7 +80,8 @@ function CssBaseline(inProps) {
   const { children, enableColorScheme = false } = props;
   return (
     <React.Fragment>
-      <GlobalStyles styles={(theme) => styles(theme, enableColorScheme)} />
+      <GlobalStyles />
+      {enableColorScheme && <span className={SELECTOR} style={{ display: 'none' }} />}
```

Comment:
What’s the need for `enableColorScheme: false`?
In other words, why would anyone want the browser to show light color scheme for a dark theme?

Having a hanging `span` element is another freak (break of principle of least surprise).

Reply:
Can we do the opposite here? Only a DOM node if the prop is not set?

Reply:
Good point, done.